services/core-api/app/auth.py

```diff
@@ -42,8 +43,9 @@ def get_current_user():
     rv = getattr(g, 'current_user', None)
     if rv == None:
         email = get_user_email()
-        rv = MinespaceUser.query.unbound_unsafe().filter_by(email=email).filter_by(
-            deleted_ind=False).first()
+        username = get_user_username()
+        rv = MinespaceUser.query.unbound_unsafe().filter(
```

Comment:
This is actually really clever. Though I think we should rename the 'email' column in the database to avoid confusion about the two types of values contained.

Reply:
I was thinking of renaming it, something generic like 'identifier' ?